doc/classes/CodeEdit.xml

```diff
@@ -64,7 +64,7 @@
 			<description>
 				Adds a comment delimiter.
 				Both the start and end keys must be symbols. Only the start key has to be unique.
-				[param line_only] denotes if the region should continue until the end of the line or carry over on to the next line. If the end key is blank this is automatically set to [code]true[/code].
+				[param line_only] defines if the region should continue until the end of the line or carry over on to the next line. If the end key is blank this is automatically set to [code]true[/code].
```

Comment:
For these I'd either use `if ... is true` and some apt change, or "controls"

Reply:
As of the time of writing I have not a clue on what's the `true` or `false` behavior or what they even mean exactly.

Reply:
Good point it is a bit cryptic, I'd say to poke who ever added this and have them help

Reply:
This was added in... 168427624ebcac64e34282eddc7856fa5de45ce6 by @Paulb23 which means it's probably time to reverse-engineer.

Reply:
Figured out, you may take a look again.